libs/mock-block-dock/src/mock-block-dock.tsx

```diff
@@ -48,8 +50,11 @@ type MockBlockDockProps = {
   blockEntityRecordId?: EntityRecordId;
   debug?: boolean;
   hideDebugToggle?: boolean;
-  initialEntities?: Entity[];
-  // initialLinkedAggregations?: LinkedAggregationDefinition[];
+  initialData?: {
+    initialTemporalAxes: QueryTemporalAxes;
```

Comment:
Does this need to be required? What would happen if we let it default to 'from 1970 to now'? What would happen if we didn't let people specify it at all?

Reply:
I tried thinking about the consequences of this and it's a bit complicated to decide how much it matters so I opted for making it required for now.

I did write a fair chunk of code before to try and make a best guess of a temporal axis from a given data set (e.g. `entity[]`) but it's impossible to determine the correct range from a single `Subgraph` of data. Opting to make it all time is _probably_ fine, but may not be, especially if we change decisions about invariants of the subgraph and say that `rightEntity` or `leftEntity`s don't need to exist in the edge sets, etc. because at that point those invariants might be broken because of the temporal query ranges and one might need to be able to look at the ranges to determine if that's the case.

And probably a bunch more things that I'm not sure we have capacity to think about right now.

Reply:
We can come back to this when we look at making MBD accept non-temporal data